Xfers/xfers-sdk
# Add Mixpanel for Tracking Pending Verification

## Approach
- Add `VERI_SG_ADD_14` for SG (Pending)
- Add `VERI_ID_ADD_08` for ID (Pending)
- Add `VERI_SG_GEN_05` for SG (Verification Required)
- Add `VERI_ID_GEN_05` for SG (Verification Required)

## Tested
- Check Rejected Re-Upload (Done)
- Check ID (Done)
- Check Web-to-Mobile (Done)
- Check MyInfo (Done)

## Screenshots
<img width="1440" alt="Screenshot 2019-11-11 at 6 41 20 PM" src="https://user-images.githubusercontent.com/13508274/68581204-e6238a00-04b2-11ea-8259-3674fdfdea49.png">
<img width="1440" alt="Screenshot 2019-11-12 at 2 52 43 PM" src="https://user-images.githubusercontent.com/13508274/68648728-196e2380-055c-11ea-8101-b0e5e42e992f.png">
